packages/decentraland-loader/lifecycle/manager.ts

```diff
@@ -108,7 +108,25 @@ export class LifecycleManager extends TransportBasedServer {
     }
   }
 
+  async invalidateAllScenes(coordsToInvalidate: string[] | undefined) {
+    for (const sceneId of this.sceneIdToRequest.keys()) {
+      await this.invalidateSceneAndCoords(sceneId)
+    }
+    if (coordsToInvalidate) this.notify('Scene.InvalidateByCoords', { coords: coordsToInvalidate })
+  }
```

Comment:
This method will invalidate all the scenes and if you pass it coordinates, it will also invalidate the coordinates for the parcels. That is because if we publish an scene of 2x2 in an scene of 1x1 plus 3 empty parcels, we need to invalidate not only the scenes, but also the parcel in order to download the new scene